examples/children.ts

```diff
@@ -16,19 +16,18 @@ const queryOptions = {
 }
 
 const client = new StreamClient(endpoint)
-const meta = namespace('https://cube.link/meta/')
 
 // Load cube using rdf-cube-view-query
 const cube = await new Source(endpoint).cube('https://environment.ld.admin.ch/foen/ubd0104/7')
 await cube.fetchShape()
 
 // Find dimensions which have a hierarchy
-const dimensions: MultiPointer = cube.ptr.any().has(meta.hasHierarchy)
+const dimensions: MultiPointer = cube.ptr.any().has(meta.inHierarchy)
 
 // Select hierarchy for the desired dimension by narrowing down on sh:path
 const hierarchy = dimensions
   .has(sh.path, $rdf.namedNode('https://environment.ld.admin.ch/foen/ubd0104/location'))
-  .out(meta.hasHierarchy)
+  .out(meta.inHierarchy)
```

Comment:
cc @ptbrowne

A bit of miscommunication on our part led me to introduce a `meta:hasHierarchy` predicate which was never intended for the [meta vocabulary](https://cube.link/meta/). 

Cube dimensions must exclusively use `meta:inHierarchy`. Note the inclusion of `@zazuko/vocabulary-extras/builders`, which, when used in TypeScript will prevent you from accidentally creating term URIs not defined in the vocabulary